src/Compilers/CSharp/Test/Semantic/Semantics/UseSiteErrorTests.cs

```diff
@@ -198,9 +198,9 @@ void CSharpErrors.InterfaceMethods.ParameterType2(UnavailableClass[] x) { }
     // (5,5): error CS0246: The type or namespace name 'UnavailableClass' could not be found (are you missing a using directive or an assembly reference?)
     //     UnavailableClass[] CSharpErrors.InterfaceMethods.ReturnType2() { return null; }
     Diagnostic(ErrorCode.ERR_SingleTypeNameNotFound, "UnavailableClass").WithArguments("UnavailableClass").WithLocation(5, 5),
-    // (5,54): error CS0539: 'C.ReturnType2()' in explicit interface declaration is not a member of interface
+    // (5,54): error CS9334: 'C.ReturnType2()': return type must be 'UnavailableClass[]' to match implemented member 'InterfaceMethods.ReturnType2()'
```

Comment:
If you wish, you could consider opening a follow up PR after this one, to avoid reporting a "bad implementation" error, when the implementation signature already has errors, which I believe is called out in the "CONSIDER" comment in the implementation.